src/api/asyncStorageApi.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+import { AsyncStorage } from "react-native";
+
+export const generateKey = (key: string) => `@AsyncStorage:${key}`;
+
+export const storeData = async (key: string, value: string) => {
+  try {
+    await AsyncStorage.setItem(generateKey(key), value);
+  } catch (error) {
+    console.log("Error saving data" + error);
+  }
+};
+
+export const retrieveData = async (key: string) => {
+  try {
+    const value = await AsyncStorage.getItem(generateKey(key));
+    if (value !== null) {
+      return value;
```

Comment:
Why don’t return null? This function must return at least something, in cases if it not throws errors